### DISCUSSION TITLE
API 명세서 DEMO

### DISCUSSION THREAD

**Discussion by yeon-so** [nhnacademy-be7-nuri-books/nuri-books-demo#5] [명세서]
## 예시

| 기능 | URL | Method | Request Data | Response Data | 기타 | 
| --- | --- | --- | --- | --- | --- |
| 회원가입 | /auth/sign-in | POST | {  "userName": “user1",  “userPassword”: “password1”,  “userEmail”: ”[user1@test.com](mailto:user1@test.com)”} | { “statusCode” : 201 “message” : “Created”} | 아이디 또는 이메일이 이미 존재하면 409{ “statusCode” : 409 “message” : “Failed”} |

도메인/기능/요청 or 응답/Dto
UserRegisterReqtDto

**Comment by yongtaek2** [comment]
누리는 내가 아는 API 작성자 중 제일이다.
